tests/validator/validator.cpp

```diff
@@ -1765,14 +1771,13 @@ TEST(Validator, unitEquivalenceMultiplierPrefix)
     libcellml::Variable::addEquivalence(v2, v3);
 
     validator->validateModel(m);
-    EXPECT_EQ(size_t(0), validator->errorCount());
+    EXPECT_EQ(size_t(0), validator->issueCount());
 }
 
 TEST(Validator, unitEquivalenceComplicatedNestedUnits)
 {
-    const std::vector<std::string> expectedErrors = {
-        "Variable 'pjs' has units of 'testunit13' and an equivalent variable 'pajamas' with non-matching units of 'testunit14'. The mismatch is: metre^1, multiplication factor of 10^3.",
-    };
+    const std::vector<std::string> expectedIssues = {
+        "Variable 'pjs' in component 'yousay' has units of 'testunit13' and an equivalent variable 'pajamas' in component 'wesay' with non-matching units of 'testunit14'. The mismatch is: metre^1, multiplication factor of 10^3."};
```

Comment:
Add a comma at the end of the issue string, so that ClangFormat can format this better.

Reply:
Done